### ISSUE TITLE
/sciencemesh/create-share defaults look wrong

### ISSUE THREAD

**Issue by michielbdejong** [cs3org/reva#3850]
Looking at https://github.com/cs3org/reva/blob/master/internal/http/services/sciencemesh/share.go#L167-L170 it seems that that code is a confusion between default type = dir and default role = viewer.
Not type == viewer right?

**Comment by gmgigi96** [comment]
Correct! The type is directly mapped to the `shareType` in the OCM APIs, so the possible values are `user`, `group` or `federation`. For now reva only implements the `user` type and is indeed hard coded when creating the OCM share.
